presto-main/src/test/java/com/facebook/presto/metadata/TestFunctionRegistry.java

```diff
@@ -293,6 +293,14 @@ public void testResolveFunctionForUnknown()
                 .failsWithMessage("Could not choose a best candidate operator. Explicit type casts must be added.");
     }
 
+    private FunctionRegistry createFunctionRegistry(TypeRegistry typeManager)
+    {
+        BlockEncodingManager blockEncodingManager = new BlockEncodingManager(typeManager);
+        FeaturesConfig featuresConfig = new FeaturesConfig();
+        FunctionManager functionManager = new FunctionManager(typeManager, blockEncodingManager, featuresConfig);
+        return new FunctionRegistry(typeManager, blockEncodingManager, new FeaturesConfig(), functionManager);
```

Comment:
We create multiple `FeaturesConfig`s in this method (another one is created on L299).